Classes/Domain/Repository/ImageFileReferenceRepository.php

```diff
@@ -35,29 +36,27 @@ class ImageFileReferenceRepository extends Repository
     /**
      * @var ConnectionPool
      */
-    private ConnectionPool $connectionPool;
+    private readonly ConnectionPool $connectionPool;
```

Comment:
this is for PHP 8+ only, right?

then you should use Constructor property promotion

https://stitcher.io/blog/constructor-promotion-in-php-8

Reply:
oh, dunno if this works for child classes ...